h/templates/app.html

```diff
@@ -16,6 +16,7 @@
           --><span class="provider" ng-show="auth.user">/{% raw %}{{ auth.user|persona:'provider' }}{% endraw %}</span><!--
           --><i class="h-icon-triangle"></i></span>
         <ul class="dropdown-menu pull-right" role="menu">
+          <li share-this-page="" ng-hide="isStream"></li>
```

Comment:
This markup is not inside the view. We really need this at the `AppController` scope. Or we could use a `run` block on the app module and set something on the root scope based on whether `$window.top is $window`.